src/app/settings/organization-plans.component.ts

```diff
@@ -100,6 +64,137 @@ export class OrganizationPlansComponent {
+        }
+
+        if (!this.showFree) {
+            validPlans = validPlans.filter((plan) => plan.product !== ProductType.Free);
+        }
+
+        validPlans = validPlans
+            .filter((plan) => !plan.legacyYear
+                && !plan.disabled
+                && (plan.isAnnual || plan.product === this.productTypes.Free));
+
+        return validPlans;
+    }
+
+    get selectablePlans() {
+        return this.plans.filter((plan) => !plan.legacyYear && !plan.disabled && plan.product === this.product);
+    }
+
+    additionalStoragePriceMonthly(selectedPlan: PlanResponse) {
+        if (!selectedPlan.isAnnual) {
+            return selectedPlan.additionalStoragePricePerGb;
+        }
+        return selectedPlan.additionalStoragePricePerGb / 12;
+    }
+
+    seatPriceMonthly(selectedPlan: PlanResponse) {
+        if (!selectedPlan.isAnnual) {
+            return selectedPlan.seatPrice;
+        }
+        return selectedPlan.seatPrice / 12;
+    }
+
+    additionalStorageTotal(plan: PlanResponse): number {
+        if (!plan.hasAdditionalStorageOption) {
+            return 0;
+        }
+
+        return plan.additionalStoragePricePerGb * Math.abs(this.additionalStorage || 0);
+    }
+
+    seatTotal(plan: PlanResponse): number {
+        if (!plan.hasAdditionalSeatsOption) {
+            return 0;
+        }
+
+        return plan.seatPrice * Math.abs(this.additionalSeats || 0);
+    }
+
+    get subtotal() {
+        let total = this.selectedPlan.basePrice;
+        if (this.selectedPlan.hasAdditionalSeatsOption && this.additionalSeats) {
+            total += this.seatTotal(this.selectedPlan);
+        }
+        if (this.selectedPlan.hasAdditionalStorageOption && this.additionalStorage) {
+            total += this.additionalStorageTotal(this.selectedPlan);
+        }
+        if (this.selectedPlan.hasPremiumAccessOption && this.premiumAccessAddon) {
+            total += this.selectedPlan.premiumAccessOptionPrice;
+        }
+        return total;
+    }
+
+    // TODO: Wire up to chosen API
+    get salesTax() {
+        return 0;
+    }
+
+    get total() {
+        return this.subtotal + this.salesTax || 0;
+    }
+
+    changedProduct() {
+        this.plan = this.selectablePlans[0].type;
+        if (!this.selectedPlan.hasPremiumAccessOption) {
+            this.premiumAccessAddon = false;
+        }
+        if (!this.selectedPlan.hasAdditionalStorageOption) {
+            this.additionalStorage = 0;
+        }
+        if (!this.selectedPlan.hasAdditionalSeatsOption) {
+            this.additionalSeats = 0;
+        } else if (!this.additionalSeats && !this.selectedPlan.baseSeats &&
+            this.selectedPlan.hasAdditionalSeatsOption) {
+            this.additionalSeats = 1;
+        }
+    }
+
+    changedOwnedBusiness() {
+        if (!this.ownedBusiness || this.selectedPlan.canBeUsedByBusiness) {
+            return;
+        }
+        this.plan = PlanType.TeamsMonthly;
+    }
+
+    changedCountry() {
```

Comment:
this method can also control sales tax re-calculation OR at least visibility of the Sales Tax element.

Reply:
I don't think this method can control re-calculation, that would make sales tax calculations go stale and be inaccurate with seat/storage/plan changes. 

I am concerned about the network usage that goes in to recalculating sales tax, especially considering we are going to our server and then going to whatever API is selected for computation. The are a lot of fields impacting sales tax amount. Recalculation is going to be needed on pretty much every change event in the form. Should we consider moving the subtotal/tax/total information to a separate confirmation screen?

Reply:
@addisonbeck , Fair enough, which is why sales tax was a stretch goal, let's scrap it for now and we'll revisit after SSO and Plan Updates are delivered/live! Thanks Addison!

Reply:
Took it all out